README.md

```diff
@@ -72,8 +72,9 @@ We are actively working to verify compatibility with these versions. If you enco
 After installing the npm package, run the following command in the `ios` directory of your React Native project.
 Install [Cocoapods](https://cocoapods.org/) if you have not already.
 ```sh
-pod install --repo-update
+pod install
```

Comment:
Hmm, my understanding here is these steps are for someone setting up the first, in which case their spec won't have our repo and `pod install` would do the job. 

For folks who already have our repo and are updating to a newer version of our RN SDK, they would encounter two cases - 
1. Our `KlaviyoSwift` SDK version has NOT changed in which case `pod install` would continue to work.
2. `KlaviySwift` SDK version has changed in which case their podfile.lock file is still locked to the older version and they can update our specific pod and then run pod install.  

For the last case, we can either include `pod update KlaviyoSwift` as a prerequisite before running `pod install` but for the first install case this may fail so I feel what I currently have here makes sense for all the cases. Let me know if I'm missing something.

[This](https://guides.cocoapods.org/using/pod-install-vs-update.html) guide has some more info on when to use install over update.

![image](https://github.com/klaviyo/klaviyo-react-native-sdk/assets/118314354/d0763e44-db67-4e61-b67e-977fb7872025)

Reply:
OK I think your assessment is right, and I also don't generally see `--repo-update` on competitors instructions. Besides which, when --repo-update is needed, the error message is quite clear so I don't think we need to include it.